### PR TITLE
7.0 hr employee product

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This is a backport to 7.0 of the module 'hr_employee_product' that has been proposed for 8.0 here: https://github.com/OCA/hr-timesheet/pull/35

### CODE REVIEW

File: hr_employee_product/model/hr.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    def on_change_unit_amount(self, cr, uid, id, prod_id, unit_amount,
+                              company_id, unit=False, journal_id=False,
+                              context=None):
+        return super(HrAnalyticTimesheet, self).on_change_unit_amount(
+            cr, SUPERUSER_ID, id, prod_id, unit_amount, company_id, unit=False,
+            journal_id=False, context=context)
+
+    def _getEmployeeProduct(self, cr, uid, context=None):
+        return super(HrAnalyticTimesheet, self)._getEmployeeProduct(
+            cr, uid, context=context)
```
Comment (pedrobaeza) [inline hr_employee_product/model/hr.py]: Why don't you use here SUPERUSER_ID?

**Comment by JordiBForgeFlow** [comment]
oops! Fixed.

**Comment by mdietrichc2c** [comment]
:+1:

**Comment by charbeljc** [comment]
Missing lxml dependency on travis build

```
  File "/home/travis/OCB-7.0/openerp/osv/orm.py", line 61, in <module>

    from lxml import etree

ImportError: No module named lxml
```

**Comment by mdietrichc2c** [comment]
@charbeljc the fix for `lxml`was done on the main 7.0 branch; this Travis should be relaunched.

**Comment by mdietrichc2c** [comment]
Actually, re-running Travis showed a PEP8 error: `./hr_employee_product/model/hr.py:22:1: F401 'fields' imported but unused` .

Also, I saw the `lxml` error, but only on the first process. Rebasing on 7.0 would be a good idea.

**Comment by feketemihai** [comment]
@jbeficent I will close this one since long time without activity, if still need it please reopen the PR.

**Pull request closed without merging**